fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyLockDiag.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (C) Copyright IBM Corp. 2019
```

Comment:
```suggestion
 * (C) Copyright IBM Corp. 2020
```

Please fix your copyright headers

Reply:
The template is this: ```
/*
 * (C) Copyright IBM Corp. 2020
 *
 * SPDX-License-Identifier: Apache-2.0
 */
```
not sure where the 2019 is coming from